charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```diff
@@ -8,91 +8,100 @@ data:
   config.yaml: |-
     auth:
       require_signing: {{ .Values.config.auth.require_signing }}
-      scheme: {{ .Values.config.auth.scheme }} 
+      scheme: {{ .Values.config.auth.scheme }}
       signing_disabled: {{ .Values.config.auth.signing_disabled }}
-      admin_key: ""
+      admin_key: {{ .Values.config.auth.admin_key }}
```

Comment:
we're setting this value as environment variable so, we don't need it here.
Please revert this change and add a comment that we're setting it as environment variable.